drivers/hmis_csv_twenty_twenty_four/app/models/hmis_csv_twenty_twenty_four/exporter/export_concern.rb

```diff
@@ -41,6 +41,11 @@ def self.replace_blank(row, hud_field:, default_value:)
       row
     end
 
+    def self.replace_newlines(row, hud_field:)
+      row[hud_field] = row[hud_field].gsub("\n", ' ') if row[hud_field].present?
```

Comment:
Does it make sense to replace them with something other than a space?  I assume the receiving entity won't care much, but if we used something like `|` then it could be restored to a new line.  That said, using a space means it's still readable.

Reply:
nvm, well, it's already quoted. IDK why we can't just have it export like `"foo\nbar"`

Reply:
I would support `"foo\nbar"`.  Not sure if that's possible or not.  The CSV processor may be converting it?